cookbooks/bcpc-hadoop/recipes/hbase_repl.rb

```diff
@@ -0,0 +1,19 @@
+#
+# To enable/disable replication peers
+#
+if (node["bcpc"]["hadoop"]["hbase"]["repl"]["enabled"])
+  bash "create_hbase_repl_peer" do
```

Comment:
Will it a better idea to use [HBase-Stargate](https://github.com/greglu/hbase-stargate)?